src/main/java/com/bookshop/dto/book/BookDto.java

```diff
@@ -2,19 +2,28 @@
 
 import io.swagger.v3.oas.annotations.media.Schema;
 import java.math.BigDecimal;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
 
-public record BookDto(
-        Long id,
-        @Schema(example = "title")
-        String title,
-        @Schema(example = "author")
-        String author,
-        @Schema(example = "978-1569319017")
-        String isbn,
-        @Schema(example = "12.99")
-        BigDecimal price,
-        @Schema(example = "some description of the book")
-        String description,
-        @Schema(example = "imageHolder.com\\url")
-        String coverImage) {
+@Data
+public class BookDto {
```

Comment:
Please choose approach use class with Data annotation or record keyword and be consistent